Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Prefabs/HolographicButton.prefab

```diff
@@ -409,7 +410,124 @@ MonoBehaviour:
       m_Calls: []
     m_TypeName: UnityEngine.Events.UnityEvent, UnityEngine.CoreModule, Version=0.0.0.0,
       Culture=neutral, PublicKeyToken=null
-  Events: []
+  Events:
```

Comment:
How are these changes to add audio events related to adding the ability to respond to grip?

Reply:
I was updating the scene to include more info on the panel and looking at documented existing functionality better, we have an audio event that was documented but it was never demoed, now that we have MRTK audio files, I added and audio event to the holographic button, and took a snapshot to replace the one in the docs. This could be a separate PR, it just felt weird to revert it.